website/src/components/toc.tsx

```diff
@@ -60,6 +61,31 @@ const CollectionTOC = () => {
           </Fragment>
         ) : null
       )}
+      <div className={css.noticeText}>
```

Comment:
I think maybe instead of putting the Creative Commons License here we could put it at the footer of the pages? To be more consistent with the DAILP pages

Reply:
Im ok adding it to the CWKW footer if @LN7-cyber is!